src/main/webapp/css/user_style.css

```diff
@@ -0,0 +1,83 @@
+.main {
+    text-align: center;
+    padding: 0px 10px;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+.column {
+    display: inline-block;
+    width: 45%;
+}
+
+.autocomplete {
+    position: relative;
+    display: inline-block;
+}
+
+input {
+    border: 1px solid transparent;
+    background-color: #f1f1f1;
+    padding: 10px;
+    font-size: 16px;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+input[type=text] {
+    background-color: #f1f1f1;
+    width: 100%;
+}
+
+input[type=submit] {
+    background-color: #4285F4;
+    color: #fff;
+}
+
+.autocomplete-items {
+    border: 1px solid #d4d4d4;
+    border-top: none;
+    z-index: 99;
+    top: 100%;
+    max-height: 300px;
+    overflow: scroll;
+    text-align: left;
+}
+
+.autocomplete-items div {
+    padding: 10px;
+    cursor: pointer;
+    border-bottom: 1px solid #d4d4d4;
+}
+
+.autocomplete-items div:hover {
+    background-color: #e9e9e9;
+}
+
+.autocomplete-active {
+    background-color: #4285F4;
+    color: #ffffff;
+}
+
+.setting {
+    margin-top: 20%;
```

Comment:
I don't think this does what you think it does. Percentage-based margins are based on the container's width! Are you trying to make this responsive with height? Or would a pixel-based margin work better?